lib/diplomat/configuration.rb

```diff
@@ -18,11 +18,13 @@ def self.parse_consul_addr
     # Override defaults for configuration
     # @param url [String] consul's connection URL
     # @param acl_token [String] a connection token used when making requests to consul
+    # @param dc [String] consul's datacenter to use as default
```

Comment:
Please specify somewhere that if nil -> fallback on agent's default DC and specify that using DC in any endpoint overrides this setting